pkg/config/shared.go

```diff
@@ -99,6 +106,7 @@ type HTTP struct {
 	RedirectToHTTPS bool             `name:"redirect-to-tls" description:"Redirect HTTP requests to HTTPS"`
 	Static          HTTPStaticConfig `name:"static"`
 	Cookie          Cookie           `name:"cookie"`
+	CSRF            CSRF             `name:"csrf"`
```

Comment:
Can we really not just re-use the existing cookie conf for CSRF cookies?

Reply:
Yeah, I guess this is unnecessary configurability. Removed.